### DISCUSSION TITLE
Save tar_visnetwork as an SVG or PDF

### DISCUSSION THREAD

**Discussion by sirusb** [ropensci/targets#867] [Help] [answered]
Hi guys, 

Thank you very much for the great package, it helped me automate and parallelize a lot of bioinformatics pipelines. 

It would be great though if we could save the tar_visnetwork as an SVG or HTML page. One trick is to create a markdown file and knitting it into an HTML page. but it would be great it is was a function call instead. it is helpful to keep track of the running time of each analysis.

**Comment by wlandau** [comment, marked as answer]
`tar_visnetwork()` returns an HTML widget object, and after that point we enter `visNetwork`/`htmlwidgets` territory. `visNetwork::visSave()` saves the object as an HTML file. And from https://github.com/ramnathv/htmlwidgets/issues/95, it looks like SVG is a work in progress. `webshot` can save to PNG, but it often looks low-res. For a completely static graph, `tar_mermaid()` is an alternative: https://github.com/ropensci/targets/pull/802

**Reply by sirusb** [reply]
Amazing! Thank you very much!

**Comment by ginolhac** [comment]
Of note, `mermaid` seems to be natively rendered in **quarto** https://mine-cetinkaya-rundel.github.io/quarto-tip-a-day/posts/21-diagrams/
Promising with **quarto** support in `{targets}`!

**Reply by sirusb** [reply]
Thanks @ginolhac, yeah indeed I am exploring Quarto recently, I will give it a try.
